pybullet_zmq/run.sh

```diff
@@ -0,0 +1,12 @@
+#!/usr/bin/env bash
+
+docker run \
+	   -it \
+	   -e DISPLAY=$DISPLAY \
+	   -h $HOSTNAME \
+	   --net host \
+	   -v /tmp/.X11-unix:/tmp/.X11-unix \
+	   -v $HOME/.Xauthority:/home/ros/.Xauthority \
+	   aica-technology/zmq-simulator
```

Comment:
```suggestion
aica-docker interactive aica-technology/zmq-simulator --net host --no-hostname
```
would be easier

Reply:
How do I approve and close the request?

Reply:
You marked the conversation as resolved. As I am the author of the PR, you would say you agree with this change by commenting here, then I commit and then you can give your final review